ignite/src/test/java/org/hibernate/ogm/datastore/ignite/test/query/nativequery/OscarWildePoem.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.datastore.ignite.test.query.nativequery;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import javax.persistence.ElementCollection;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.NamedNativeQueries;
+import javax.persistence.NamedNativeQuery;
+
+@Entity
+//@todo fix problem with rename cache
+//@Table(name = OscarWildePoem.TABLE_NAME)
+
+
+@NamedNativeQueries({
+		@NamedNativeQuery(name = "PortiaOscarWildePoem", query = "select _key,_val from OscarWildePoem  where  name='Portia' and author='Oscar Wilde'", resultClass = OscarWildePoem.class),
+		@NamedNativeQuery(name = "OscarWildePoemWithParameters", query = "select _key,_val from OscarWildePoem  where  name=?1 and author=?2", resultClass = OscarWildePoem.class)
+})
+public class OscarWildePoem {
+
+	//public static final String TABLE_NAME = "WILDE_POEM";
```

Comment:
If it's not needed for the test, why don't you delete it?

Reply:
hm .... I think I have to develop additional test for test support annotation @Table .  
Okey. I will clear comments and develop additional test.